src/actions/peers.js

```diff
@@ -1,12 +1,12 @@
-import panacea from '@medibloc/panacea-js';
+import { Client } from '@medibloc/panacea-js';
```

Comment:
This is because the new `panacea-js` doesn't use `export default ...` stuff.

If it's imported by `import panacea from ...`, Webpack converts it as something like `_panaceaJ2.default.XXX` which doesn't work at all